elasticsearch.go

```diff
@@ -4,7 +4,7 @@ import (
 	"encoding/json"
 	"github.com/buger/elastigo/api"
 	"github.com/buger/elastigo/core"
-	"github.com/buger/gor/proto"
```

Comment:
changing all of these imports to point to our fork might make this change difficult to PR on upstream gor. I think we also keep the original import statements in other forked repos (e.g. drone). Would it be possible to keep the original imports?